internal/storage/storage_handle_test.go

```diff
@@ -23,254 +23,257 @@ import (
 
-	AssertNe(nil, bucketHandle)
-	AssertEq(TestBucketName, bucketHandle.bucketName)
+	assert.NotNil(testSuite.T(), bucketHandle)
+	assert.Equal(testSuite.T(), TestBucketName, bucketHandle.bucketName)
 }
 
-func (t *StorageHandleTest) TestBucketHandleWhenBucketDoesNotExistWithEmptyBillingProject() {
-	storageHandle := t.fakeStorage.CreateStorageHandle()
+func (testSuite *StorageHandleTest) TestBucketHandleWhenBucketDoesNotExistWithEmptyBillingProject() {
+	storageHandle := testSuite.fakeStorage.CreateStorageHandle()
 	bucketHandle := storageHandle.BucketHandle(invalidBucketName, "")
 
-	AssertEq(nil, bucketHandle.Bucket)
+	assert.Nil(testSuite.T(), bucketHandle.Bucket)
 }
 
-func (t *StorageHandleTest) TestBucketHandleWhenBucketExistsWithNonEmptyBillingProject() {
-	storageHandle := t.fakeStorage.CreateStorageHandle()
+func (testSuite *StorageHandleTest) TestBucketHandleWhenBucketExistsWithNonEmptyBillingProject() {
+	storageHandle := testSuite.fakeStorage.CreateStorageHandle()
 	bucketHandle := storageHandle.BucketHandle(TestBucketName, projectID)
 
-	AssertNe(nil, bucketHandle)
-	AssertEq(TestBucketName, bucketHandle.bucketName)
+	assert.NotNil(testSuite.T(), bucketHandle)
+	assert.Equal(testSuite.T(), TestBucketName, bucketHandle.bucketName)
 }
 
-func (t *StorageHandleTest) TestBucketHandleWhenBucketDoesNotExistWithNonEmptyBillingProject() {
-	storageHandle := t.fakeStorage.CreateStorageHandle()
+func (testSuite *StorageHandleTest) TestBucketHandleWhenBucketDoesNotExistWithNonEmptyBillingProject() {
+	storageHandle := testSuite.fakeStorage.CreateStorageHandle()
 	bucketHandle := storageHandle.BucketHandle(invalidBucketName, projectID)
 
-	AssertEq(nil, bucketHandle.Bucket)
+	assert.Nil(testSuite.T(), bucketHandle.Bucket)
 }
 
-func (t *StorageHandleTest) TestNewStorageHandleHttp2Disabled() {
+func (testSuite *StorageHandleTest) TestNewStorageHandleHttp2Disabled() {
 	sc := storageutil.GetDefaultStorageClientConfig() // by default http1 enabled
 
-	t.invokeAndVerifyStorageHandle(sc)
-}
-
-func (t *StorageHandleTest) TestNewStorageHandleHttp2Enabled() {
-	sc := storageutil.GetDefaultStorageClientConfig() // by default http1 enabled
+	handleCreated, err := NewStorageHandle(context.Background(), sc)
 
-	t.invokeAndVerifyStorageHandle(sc)
+	assert.Nil(testSuite.T(), err)
+	assert.NotNil(testSuite.T(), handleCreated)
 }
 
-func (t *StorageHandleTest) TestNewStorageHandleHttp2EnabledAndAuthEnabled() {
+func (testSuite *StorageHandleTest) TestNewStorageHandleHttp2EnabledAndAuthEnabled() {
 	sc := storageutil.GetDefaultStorageClientConfig()
 	sc.ClientProtocol = mountpkg.HTTP2
 	sc.AnonymousAccess = false
 
 	handleCreated, err := NewStorageHandle(context.Background(), sc)
 
-	AssertNe(nil, err)
-	ExpectThat(err, oglematchers.Error(oglematchers.HasSubstr("no such file or directory")))
-	AssertEq(nil, handleCreated)
+	assert.NotNil(testSuite.T(), err)
+	assert.True(testSuite.T(), strings.Contains(err.Error(), "no such file or directory"))
```

Comment:
you can either use https://pkg.go.dev/os#IsNotExist or make use of assert.Contains or assert.ErrorContains https://pkg.go.dev/github.com/stretchr/testify/assert#hdr-Assertions